drools-benchmarks/pom.xml

```diff
@@ -5,61 +5,78 @@
   <parent>
     <artifactId>kie-benchmarks</artifactId>
     <groupId>org.kie</groupId>
-    <version>7.68.0-SNAPSHOT</version>
+    <version>8.19.0-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>org.drools</groupId>
   <artifactId>drools-benchmarks</artifactId>
 
+  <properties>
+    <version.org.drools>8.19.0-SNAPSHOT</version.org.drools>
+    <version.kie.server.api>7.68.0-SNAPSHOT</version.kie.server.api>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.drools</groupId>
       <artifactId>drools-core</artifactId>
+      <version>${version.org.drools}</version>
```

Comment:
Is it possible to use bom in dependency management to not specify the versions?

Reply:
I think so. I made it rough so I could compile it quickly. If we go ahead with this change, a refactoring is welcome here.